src/radix_connect/p2p_links/p2p_link.rs

```diff
@@ -1,5 +1,7 @@
 use crate::prelude::*;
 
+json_data_convertible!(P2PLink);
+
 /// A client the user have connected P2P with, typically a
 /// WebRTC connections with a DApp, but might be Android or iPhone
 /// client as well.
```

Comment:
is this doc still accurate? maybe it can be updated a bit?

Reply:
links to any Confluence Page with CAP36 would be good. See how I've done it e.g. here with links:

https://github.com/radixdlt/sargon/blob/39ad077a83b50b93b322a00319398d3a47835d80/src/profile/v100/address/account_address.rs#L4-L26